snapshots/biodiversity/2024-01-25/cherry_blossom.xls.dvc

```diff
@@ -0,0 +1,46 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Cherry Blossom Full Bloom Dates in Kyoto, Japan
+    description: |-
+      A historical time series of peak cherry blossom bloom data from Kyoto, Japan. The timing of peak cherry blossom is linked to weather and climate factors. Warmer weather, due to climate change, is pulling peak cherry blossom earlier in the year.
+
+      Yasuyuki Aono compiled this data based on multiple sources:
+
+            - Taguchi,T. (1939) Climatic change in historical time in Japan J. Mar. Meteorol. Soc. 19 217–27
+
+            - Sekiguchi,T. (1969) The historical dates of Japanese cherry festival since the 8th century and her climatic changes Tokyo Geog. Pap. 13 175–90
+
+            - Aono,Y. and Omoto,Y. (1994) Estimation of temperature at Kyoto since 11th century using flowering data of cherry tree in old documents J. Agric. Meteorol. 49 263–72
+
+            - Aono,Y. and Kazui,K. (2008) Phenological data series of cherry tree flowering in Kyoto, Japan, and its application to reconstruction of springtime temperatures since the 9th century Int. J. Climatol. 28 905–14
+
+            - Aono,Y. and Saito,S. (2010) Clarifying springtime temperature reconstructions of the medieval period by gap-filling the cherry blossom phenological data series at Kyoto, Japan Int. J. Biometeorol. 54 211–9
+
+            Data for 812-2021 is available here - http://atmenv.envi.osakafu-u.ac.jp/aono/kyophenotemp4/
+
+            Data for 2022 & 2023 full blossom is gathered from personal communication with Yasuyuki Aono.
+
+    date_published: 2021-03-06
+    # Citation
+    producer: Yasuyuki Aono
+    citation_full: |-
+      - Aono and Kazui (2008) International Journal of Climatology, 28, 905-914
+      - Aono and Saito (2010) International Journal of Biometeorology, 54, 211-219
+    attribution_short: Aono
+    # Files
+    url_main: http://atmenv.envi.osakafu-u.ac.jp/aono/kyophenotemp4/
+    url_download: http://atmenv.envi.osakafu-u.ac.jp/omu-content/uploads/sites/1215/2015/10/KyotoFullFlower7.xls
+    date_accessed: 2024-01-25
+
+    # License
+    license:
+      name: CC BY 4.0
```

Comment:
if they don't provide a link to the license, maybe you can just add a generic link https://creativecommons.org/licenses/by/4.0/deed.en